HelloWorld sample for dotnetcore 3.0 doesn't work

I'm trying to run BuildAndRun.ps1 for orleans\samples\3.0\HelloWorld and it fails with the following output 

PS C:\Users\shreyagr\source\repos\orleans\Samples\3.0\HelloWorld> .\BuildAndRun.ps1
  Restore completed in 74.22 ms for C:\Users\shreyagr\source\repos\orleans\Samples\3.0\HelloWorld\src\HelloWorld.Interfaces\HelloWorld.Interfaces.csproj.
  Restore completed in 89.26 ms for C:\Users\shreyagr\source\repos\orleans\Samples\3.0\HelloWorld\src\HelloWorld.Grains\HelloWorld.Grains.csproj.
  Restore completed in 89.26 ms for C:\Users\shreyagr\source\repos\orleans\Samples\3.0\HelloWorld\src\SiloHost\SiloHost.csproj.
  Restore completed in 89.27 ms for C:\Users\shreyagr\source\repos\orleans\Samples\3.0\HelloWorld\src\OrleansClient\OrleansClient.csproj.
Microsoft (R) Build Engine version 16.4.0+e901037fe for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  HelloWorld.Interfaces -> C:\Users\shreyagr\source\repos\orleans\Samples\3.0\HelloWorld\src\HelloWorld.Interfaces\bin\Debug\netstandard2.0\HelloWorld.Interfaces.dll
  HelloWorld.Grains -> C:\Users\shreyagr\source\repos\orleans\Samples\3.0\HelloWorld\src\HelloWorld.Grains\bin\Debug\netstandard2.0\HelloWorld.Grains.dll
  OrleansClient -> C:\Users\shreyagr\source\repos\orleans\Samples\3.0\HelloWorld\src\OrleansClient\bin\Debug\netcoreapp2.1\OrleansClient.dll
Program.cs(30,26): error CS1061: 'ISiloBuilder' does not contain a definition for 'AddAzureBlobGrainStorage' and no accessible extension method 'AddAzureBlobGrainStorage' accepting a first argument of type 'ISiloBuilder' could be found (are you missing a using directive or an assembly reference?) [C:\Users\shreyagr\source\repos\orleans\Samples\3.0\HelloWorld\src\SiloHost\SiloHost.csproj]

Build FAILED.

Program.cs(30,26): error CS1061: 'ISiloBuilder' does not contain a definition for 'AddAzureBlobGrainStorage' and no accessible extension method 'AddAzureBlobGrainStorage' accepting a first argument of type 'ISiloBuilder' could be found (are you missing a using directive or an assembly reference?) [C:\Users\shreyagr\source\repos\orleans\Samples\3.0\HelloWorld\src\SiloHost\SiloHost.csproj]
    0 Warning(s)
    1 Error(s)

--- a/Samples/3.0/HelloWorld/src/SiloHost/SiloHost.csproj
+++ b/Samples/3.0/HelloWorld/src/SiloHost/SiloHost.csproj
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <IsPackable>false</IsPackable>
   </PropertyGroup>
@@ -16,6 +16,7 @@
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="3.0.0" />
     <PackageReference Include="Microsoft.Orleans.OrleansRuntime" Version="3.0.0" />
     <PackageReference Include="Microsoft.Orleans.OrleansProviders" Version="3.0.0" />
+    <PackageReference Include="Microsoft.Orleans.Persistence.AzureStorage" Version="3.0.0" />
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\HelloWorld.Grains\HelloWorld.Grains.csproj" />

